apps/site/assets/ts/stop/components/PillLink.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactElement } from "react";
+
+export const bgClass = (color: string | undefined): string =>
+  color ? `u-bg--${color}` : "";
+
+const PillLink = ({
+  displayText,
+  linkText,
+  backgroundColor,
+  optionalCSS = "",
+  externalLink
+}: {
+  displayText: string;
+  linkText: string;
```

Comment:
I think a name that clarifies its meaning as a URL or path would be better here, maybe `linkURL` or even `linkTo`?